source/reference/mongosync.txt

```diff
@@ -176,6 +176,38 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-oidc:
+
+Microsoft Azure Managed Identities Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+To use Microsoft Azure Managed Identities for federated authentication,
+define these environment variables:
+
+.. option:: AZURE_TENANT_ID <id>
+
+   Sets the Azure tenant identifier. Example:
```

Comment:
Why are these listed here? I think they should probably be constrained to the oidc docs page above

Reply:
azure managed identities is currently *only* supported by mongosync

Reply:
@rohshar @pmeredit so, leave in and as written? Cheers.